### PR TITLE
Remove Go content

### PR DESCRIPTION
**Pull request by huwd**
One for folks with more GDS history than me to 👀 but as I reviewed around these changes felt right to me.

### CODE REVIEW

**Comment by huwd** [comment]
Question... does removing it here mean we should remove the styleguides and links?

https://gds-way.digital.cabinet-office.gov.uk/manuals/programming-languages.html
&
https://gds-way.digital.cabinet-office.gov.uk/manuals/programming-languages/go.html

In my head:
Argument for - Clean up, why maintain redundant pages when we don't have to. These pages are unlikely to see much use, should they have a future need they can be resurrected from the git history.

Argument against - It's still used in GOV.UK's Router... so might be useful to see how to refactor / write fixes in good style. Plus there's a good banner at the top saying it's not core, so limited risk someone gets excited and starts using it, better chance they search find this and find out there's a head of tech to speak to?

File: source/standards/programming-languages.html.md.erb [hunk 1]
```diff
@@ -77,30 +77,26 @@ production.
...
+#### Go
 
 Go is no longer a core backend development language in GDS.
 
 The only Go service currently in production operation is the [GOV.UK
-router][router], and it is the core language for [Cloud
-Foundry](https://www.cloudfoundry.org/), which GOV.UK PaaS uses, although
-GOV.UK PaaS is being [decommissioned][pass-decom]. As such, the knowledge and
-experience of building and running services in Go is small and decreasing.
+Foundry](https://www.cloudfoundry.org/), which was used by GOV.UK PaaS.
```
Comment (paroxp) [inline source/standards/programming-languages.html.md.erb:93]: You may want to keep the precious copy? Router is still used, right? And CloudFoundry is no-mo, along the paas-decom.

Comment (timblair) [reply source/standards/programming-languages.html.md.erb:93]: I'm not sure what's happened with this commit, but this paragraph makes no sense. I'm assuming this is supposed to say GOV.UK Router, and there's no need to mention PaaS:

> the GOV.UK Foundry, which was used by GOV.UK PaaS

File: source/standards/programming-languages.html.md.erb [hunk 2]
```diff
@@ -79,10 +79,24 @@ production.
...
 of skills in GDS. Do not use Scala for new projects.
 
+#### Go
+
+Go is no longer a core backend development language in GDS.
+
+The only Go service currently in production operation is the [GOV.UK
+Foundry](https://www.cloudfoundry.org/), which was used by GOV.UK PaaS.
+
+PaaS has now been [decommissioned][pass-decom].
```
Comment (timblair) [inline source/standards/programming-languages.html.md.erb:95]: PaaS has not yet been completely decommissioned. While it's no longer available for general use, it's still being operated within GDS, and used by DBT.

**Review by louzoid-gds — COMMENTED** [review]
I think I'm mostly minded to leave the guidance on Go as it presently stands as GOV.UK PaaS is still in use and supported. We are also already making it pretty clear that we shouldn't be using it for new things. 
Thoughts?

**Comment by huwd** [comment]
cool cool closing down

**Pull request closed without merging**